RecoEcal/EgammaClusterAlgos/src/EndcapPiZeroDiscriminatorAlgo.cc

```diff
@@ -10,52 +10,30 @@
 
 using namespace std;
 
+namespace {
+  constexpr int Nfiles_EE = 5;
+}
+
 EndcapPiZeroDiscriminatorAlgo::EndcapPiZeroDiscriminatorAlgo(double stripEnergyCut, int nStripCut, const string& path) :
-   preshStripEnergyCut_(stripEnergyCut),  preshSeededNstr_(nStripCut), pathToFiles_(path)
+   preshStripEnergyCut_(stripEnergyCut),  preshSeededNstr_(nStripCut)
 {   
    
      // Read all Weight files
-     Nfiles_EB = 5;
-     Nfiles_EE = 5;
+     const int Nfiles_EB = 5;
      string file_pt[5] = {"20","30","40","50","60"};
      string file_barrel_pt[5] = {"20","30","40","50","60"};   
```

Comment:
```
     string file_barrel_pt[Nfiles_EB] = {"20","30","40","50","60"}; 
```

Reply:
I can make the two suggested changes, however I was trying to make as few changes to the original as possible.

A better change would be to use an std::array and get rid of the need for `Nfiles_EB` all together, since it is now just used to iterate over the container.

Let me know which you'd prefer.

Reply:
Whatever can get rid of the hard coded "5"'s is fine
(I know, it doesn't strictly belong to what you updated: but since you already cleaned other parts of the code, why not doing it also here...)